dcr/azure-pipelines.yml

```diff
@@ -0,0 +1,102 @@
+parameters:
```

Comment:
This file is the entry point into the pipeline code, you can follow the flow from here